app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts, foreign_key: 'author_id'
+  has_many :likes
```

Comment:
- [Optional] I mark this comment optional because the first reviewer didn’t point it but it is one of the parts of the project. According to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) `comments` and `likes` tables has foreign_key `author_id` as well, kindly fix that 🙏🏾 🙏🏾

Reply:
@cynthiainga the ERdiagram l had is quite different from urs mine has the user_id as the comments and likes foreign key